.gitignore

```diff
@@ -74,4 +74,6 @@ sonar-project.properties
 /gradle.properties
 /volume/
 
-.trivyignore
\ No newline at end of file
+.trivyignore
+
+spike-env
```

Comment:
Used for a virtual environment for running the python code locally when modifying it, not to be committed